.github/workflows/pre-commit.yml

```diff
@@ -8,6 +8,10 @@ jobs:
   pre-commit:
     runs-on: ubuntu-latest
     steps:
+    - name: Install copyright check
+      shell: bash
+      run: |
+        pip install git+https://github.com/espressif/check-copyright.git@master
```

Comment:
Could you try removing these 4 lines? Since the copyright check is part of the pre-commit, the package will be installed by pre-commit framework automatically into its own virtual environment.

Reply:
I added this, because the CI failed without it (it was only in pre-commit)

Reply:
Probably you were installing check-copyright==1.0.2? It should work with 1.0.3 without these 4 lines.